Targets/AppUIKit/Sources/wallet/WalletHomeView.swift

```diff
@@ -40,29 +40,61 @@ struct WalletHomeHeader: View {
-                    Section {
-                        ForEach(credentials, id: \.self.id) { credential in
-                            AchievementCredentialItem(
-                                rawCredential: credential.rawCredential,
-                                onDelete: {
-                                    _ = CredentialDataStore.shared.delete(id: credential.id)
-                                    self.credentials = CredentialDataStore.shared.getAllCredentials()
-                                }
+            if !credentials.isEmpty {
+                ZStack {
+                    ScrollView(.vertical, showsIndicators: false) {
+                        Section {
+                            ForEach(credentials, id: \.self.id) { credential in
+                                AchievementCredentialItem(
+                                    rawCredential: credential.rawCredential,
+                                    onDelete: {
+                                        _ = CredentialDataStore.shared.delete(id: credential.id)
+                                        self.credentials = CredentialDataStore.shared
+                                            .getAllCredentials()
+                                    }
+                                )
+                            }
+                            //                    ForEach(vcs, id: \.self) { vc in
+                            //                        GenericCredentialListItem(vc: vc)
+                            //                    }
+                            //                    ShareableCredentialListItem(mdoc: mdocBase64)
+                        }
+                        .padding(.bottom, 50)
+                    }
+                    .padding(.top, 20)
+
+                    VStack {
+                        Spacer()
+                        Button(action: {
+                            path.append(ScanOID4VP())
+                        }) {
+                            HStack {
+                                Image("QRCodeReader")  // Ensure image is added to assets
+                                    .resizable()
+                                    .scaledToFit()
+                                    .frame(width: 24, height: 24)  // Adjust size as needed
+                                    .foregroundColor(.white)
+                                Text("Scan to share")
+                                    .font(.system(size: 15))
+                                    .fontWeight(.regular)
+                                    .foregroundColor(.white)
+                            }
+                            .padding(8)
+                            .frame(maxWidth: .infinity)
+                            .background(
+                                Color(
+                                    UIColor(
+                                        red: 72 / 255, green: 140 / 255, blue: 244 / 255, alpha: 1))
```

Comment:
You can create or use an existing custom color.  You have to confirm, but I guess this is the `CTAButtonBlue`